Maquetado/index.html

```diff
@@ -11,25 +11,26 @@
 
 <body>
   <div class='bookContainer'>
-    <img src='./resources/img/book-cover.png' class='bookImage' />
-    <img src='./resources/img/badge.png' class='bookBadge' />
+    <div class='bookImageContainer'>
+      <img src='./resources/img/book-cover.png' />
+    </div>
```

Comment:
You don't need to define a new div, you could just use the pseudoElemento on the 'bookContainer' div.
```suggestion
      <img src='./resources/img/book-cover.png' />
```